### PR TITLE
Updated Indie Deploy process

### PR DESCRIPTION
**Pull request by gpolumbo-broad**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +14,35 @@ Bond supports test runners: [unittest](https://docs.python.org/2/library/unittes
...
+
+[Virtualenv](https://virtualenv.pypa.io/en/stable/) is a tool that helps you manage multiple Python versions and your 
+project dependencies.  We recommend you setup Virtualenv for development and testing of Bond.
+
+1. Verify that you have Python 2.7 installed: `python --version`
+(**Note**: The name of your Python 2.7 command may be something different like `python2` if you have multiple versions 
+of Python installed)
+1. Install virtualenv: `pip install virtualenv`
+1. `cd` to the Bond root directory
+1. Setup virtualenv for Bond: `virtualenv -p python env`
```
Comment (marctalbott) [inline README.md]: "Set up"

File: README.md [hunk 2]
```diff
@@ -54,29 +82,40 @@ b) Run your local code:
...
-
-`staging` environment (branch: `staging`):
-1) Open a PR merging your `dev_tests_passed_<TIMESTAMP>_release` branch into the `staging` branch
-2) Jenkins will kick off a deploy to the `staging` environment
-
-`prod` environment (branch: `master`):
-1) Open a PR merging your `dev_tests_passed_<TIMESTAMP>_release` branch into the `master` branch
-2) Jenkins will kick off a deploy to the `prod` environment
+Deployments to non-production and production environments are performed in Jenkins.  In order to access Jenkins, you
+will need to be on the Broad network or logged onto the Broad VPN.
```
Comment (marctalbott) [inline README.md]: "logged on to"

File: README.md [hunk 3]
```diff
@@ -54,29 +82,40 @@ b) Run your local code:
...
+## Deploy to the "dev" environment
+
+A deployment to `dev` environment will be automatically triggered every time there is a commit or push to the 
+[develop](https://github.com/DataBiosphere/bond/tree/develop) branch on Github.  If you would like to deploy a different 
+branch or tag to the `dev` environment, you can do so by following the instructions below, but be aware that a new
+deployment of the `develop` branch will be triggered if anyone commits or pushes to that branch.
+
+## Deploy to non-production environments
+
+1. Log into [Jenkins](https://fc-jenkins.dsp-techops.broadinstitute.org/)
```
Comment (marctalbott) [inline README.md]: "Log in to"

File: README.md [hunk 4]
```diff
@@ -54,29 +82,40 @@ b) Run your local code:
...
+   the `SLACK_CHANNEL` that you would like to receive notifications of the deploy jobs success/failure  
+1. Click the `Build` button
+
+## Deploy to the "prod" environment
+
+Production deployments are very similar to deployments for any other environment.  The few differences are that you may 
+only deploy to Production from the prod Jenkins instance, and you are only allowed to deploy tags, not branches.
+
+1. Create a `git tag` for the commit that you want to release and push it to the [Bond repository on Github](https://github.com/DataBiosphere/bond)
+1. Log into [Prod Jenkins](https://fcprod-jenkins.dsp-techops.broadinstitute.org/)
```
Comment (marctalbott) [inline README.md]: "Log in to"

**Review by marctalbott — APPROVED** [review]
Really minor grammar nits based on [this](https://www.quickanddirtytips.com/education/grammar/in-to-or-into?page=1). Looks good otherwise.

**Review by Qi77Qi — APPROVED** [review]

**Review by dvoet — APPROVED** [review]

**Pull request merged**